src/openms/include/OpenMS/METADATA/SpectrumMetaDataLookup.h

```diff
@@ -177,15 +181,17 @@ namespace OpenMS
     };
 
     /// Constructor
-    SpectrumMetaDataLookup(): SpectrumLookup() {}
+    SpectrumMetaDataLookup(): SpectrumLookup(),
```

Comment:
newline after SpectrumLookup(), spectra_data is default initialized and needs no explicit construction here